zeth-contracts/contracts/Groth16Verifier.sol

```diff
@@ -36,27 +35,25 @@ contract Groth16Verifier {
     VerifyingKey verifyKey;
 
     event LogVerifier(string);
+    event LogVerifierUint(uint);
```

Comment:
Where do you use this event? I guess it was for a debugging purpose. Shall we keep it or do you think we ca remove it now if it is not used?

Reply:
Thanks.  Removed in latest version.